### PR TITLE
add mkb dataset

### PR DESCRIPTION
**Pull request by VasudevGupta7**
This PR will add Mann Ki Baat dataset (parallel data for Indian languages).

### CODE REVIEW

File: datasets/mkb/README.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+- Telugu
+- Tamil
+- Malayalam
+- Gujarati
+- Urdu
+- Bengali
+- Oriya
+- Marathi
+- Punjabi
+- English
```
Comment (lhoestq) [inline datasets/mkb/README.md:19]: Can you use the tagger app to fill the languages with the proper language codes ?
You can find more info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

File: datasets/mkb/mkb.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+
+        lang_pair = self.config.language_pair
+        src_tag, tgt_tag = lang_pair.split("-")
+
+        dl_dir = dl_manager.download_and_extract(_URL)
+        data_dir = os.path.join(dl_dir, f"mkb/{lang_pair}")
```
Comment (lhoestq) [inline datasets/mkb/mkb.py:127]:
```suggestion
        data_dir = os.path.join(dl_dir, "mkb", lang_pair)
```
we need that for windows users

**Review by lhoestq — APPROVED** [review]
Awesome good job ! thanks for adding this one

I left a few comments

**Comment by lhoestq** [comment]
Could you update the languages tags before we merge @VasudevGupta7 ?

**Comment by VasudevGupta7** [comment]
done.

**Comment by lhoestq** [comment]
thanks !

**Pull request merged**